packages/transport-webrtc/src/private-to-private/signaling-stream-handler.ts

```diff
@@ -96,11 +87,6 @@ export async function handleIncomingStream ({ peerConnection, stream, signal, co
       signal,
       log
     })
-
-    log.trace('recipient connected, closing signaling stream')
-    await messageStream.unwrap().unwrap().close({
-      signal
-    })
```

Comment:
This method is called from `_onProtocol` in `transport.ts` which handles closing/aborting the stream so there's no need to do it here.